pkg/crc/preflight/preflight_checks_windows.go

```diff
@@ -58,15 +58,11 @@ func checkHyperVInstalled() (bool, error) {
 		return false, errors.New("Hyper-V not installed")
 	}
 
-	// Check if Hyper-V's Virtual Machine Management Service is running
-	checkVmmsRunning := `@(Get-Service vmms).Status`
-	stdOut, _, err = powershell.Execute(checkVmmsRunning)
+	checkVmmsExists := `@(Get-Service vmms).Status`
+	_, _, err = powershell.Execute(checkVmmsExists)
```

Comment:
This call fails on my machine ("failed to find vmms service", ObjectNotFound ServiceCommandException on stderr), but `err` is `nil` nonetheless, meaning that the check is considered to be successful

Reply:
Ah right... this is what I wanted to know. So this err checking is f*****d.... :-/